SaveGameEditor/ExtrasForm.cs

```diff
@@ -47,7 +47,7 @@ private void buttonLoadHeader_Click(object sender, EventArgs e)
             comboBoxHeaderEp.Items.Clear();
             comboBoxPoint.Items.Clear();
             
-            if (_settingManager.Settings.RewindNotesShown)
+            if (!_settingManager.Settings.RewindNotesShown)
             {
                 MessageBox.Show("A few notes on the Checkpoint Rewind feature:\n\n1. It has not been extensively tested and may cause unintended consequences. Users are advised to make backups of their data before proceeding further.\n2. If you are on a mid-level checkpoint, then selecting the last item in the checkpoint list will still cause the game to start from the beginning of that point.\n3. If you want to change only the date of the save, don't touch the checkpoint list at all. To reset the state of the list, click \"Load\" again.", "Savegame Viewer", MessageBoxButtons.OK, MessageBoxIcon.Information);
```

Comment:
Please use StringBuilder to build message line-by-line using Append() and AppendLine() methods.